api/docs/code_reviews.dox

```diff
@@ -211,7 +211,13 @@ git review
 ```
 
 Do not use a force push to change the history of the shared branch!  Use a new,
-separate commit (these separate commits will all be squashed together upon merging).
+separate commit.  These separate commits will all be squashed together upon merging.
+The commit message of the final merge commit is taking from the pull request's title
+and description, which came from the initial branch commit and are unaffected by
+subsequent commits.  The pull request title and description can be edited in the web
+interface.  There is no benefit to a force push and many negatives: it makes
+reviewing much harder as it is not possible to see changes since the last review
```

Comment:
May be more accurate to say "to see the Github-generated diff of changes ..."

Reply:
Done.